Assets/BossRoom/Scripts/Shared/Net/ConnectionManagement/ClientGameNetPortal.cs

```diff
@@ -31,7 +31,7 @@ public class ClientGameNetPortal : MonoBehaviour
         /// Time in seconds before the client considers a lack of server response a timeout
         /// </summary>
         private const int k_TimeoutDuration = 10;
-        const int k_NbReconnectAttempts = 1;
+        const int k_NbReconnectAttempts = 2;
```

Comment:
From my current experiments, one reconnect attempt seems enough. Each will have a 10 sec timeout. Waiting 20 seconds to then be able to join again seems excessive.